hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeManager.java

```diff
@@ -201,8 +206,16 @@
    */
   private final boolean useDfsNetworkTopology;
 
+  private static final String IP_PORT_SEPARATOR = ":";
+
   @Nullable
   private final SlowPeerTracker slowPeerTracker;
+  private static Set<Node> slowPeers = Sets.newConcurrentHashSet();
```

Comment:
We might as well change this variable name to fix a checkstyle warning.

Reply:
Sorry, I didn't see your reply just now. I will fix it soon.